TA-Grant1101/g2g
# BE: Status track

Added Dashboard Controller to track the books lent and borrowed. 

To test
Use Get request http://localhost:8080/api/dashboard/lent/1  you should be able to see the books that are lent out by customer, and

Use Get request http://localhost:8080/api/dashboard/borrow/1  you should be able to see the books that are borrowed by customer
